pkg/issuer/venafi/client/request.go

```diff
@@ -165,7 +165,7 @@ func newVRequest(cert *x509.Certificate) *certificate.Request {
 	req.CustomFields = []certificate.CustomField{
 		{
 			Type:  certificate.CustomFieldOrigin,
-			Value: "Jetstack cert-manager",
+			Value: "cert-manager",
```

Comment:
Maybe use `util.CertManagerUserAgent` here too?

Reply:
@wallrj there was a reason from Venafi specifically why the version isn't here.... as far as i remember....

Reply:
/unhold